env.sh

```diff
@@ -43,6 +43,8 @@ elif [[  $LOGINHOST = bsc ]]; then
    STRATEGY="bsc"
 elif [[  $LOGINHOST =~ ^juwels[0-9][0-9].ib.juwels.fzj.de$ ]]; then
    STRATEGY="juwels"
+elif [[  $LOGINHOST =~ ^jwlogin[0-9][0-9].juwels$ ]]; then
+   STRATEGY="juwels"
```

Comment:
Better to keep one for `juwels`, the shorter the better :)

Reply:
I added the second juwels case to allow an easy compilation on the login nodes. But I agree, it is not necessary.